app/views/partners/campaigns/show.html.erb

```diff
@@ -67,6 +67,14 @@
     </div>
   </div>
 
+  <% if @campaign.running? %>
+    <p class="mb-5">
+      <strong>Important :</strong> Si vous avez l’impression que la campagne ne se remplit pas, ne créez pas une
+      nouvelle campagne pour les mêmes doses disponibles. <a href="/faq">Nous recontacterons par SMS tous les
+      volontaires proches de votre centre</a> s’il vous reste des doses au cours de la campagne.
+    </p>
+  <% end %>
+
```

Comment:
Maybe we could add the new paragraph inside the `<% if @campaign.running? %>` that is on line 78 ? So we don't repeat the condition twice 🙃

Reply:
Yes you're right. It's just that i wasn't sure of the status and condition. If it's only on @campaign.running, I'll add it to the condition below 🤓

Reply:
<a href="/faq"> 
URL should be generated instead